src/components/album/Album.js

```diff
@@ -30,6 +31,9 @@ const AlbumTitle = styled('h3')({
 
 function Album({ album, wishListId }) {
   const [openDialog, setOpenDialog] = useState(false);
+  const reviewsRef = React.useRef(); // Created a ref to hold the function
+  const fullWidth = true;
+  const maxWidth = 'sm';
```

Comment:
Will these be used in multiple places, or can we just inline them on the lines where they are being used for the `<Dialog>` component?